### DISCUSSION TITLE
Senior Supabase Engineer to setup backend +API point for a Flutter App

### DISCUSSION THREAD

**Discussion by vmodha** [supabase/supabase#25546] [Jobs Board]
We're creating a Flutter application for wellness that will involve live events and need someone to setup the backend and api end points for a self hosted Supabase setup.

We have:
Users, following, messaging...
Live events...
Notifications
Lots of statistics for reporting detailed information to users.

Please email me at vishal@digitalmycelium.com 

We're looking to hire immediately.

Thank you
Vishal

**Comment by Enigma-I-am** [comment]
Is this role still open? @vmodha

**Reply by vmodha** [reply]
This position has now been filled.
